src/app/pages/DashboardPage/TotalTransactions.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Card from '@mui/material/Card'
+import CardHeader from '@mui/material/CardHeader'
+import CardContent from '@mui/material/CardContent'
+import { useTheme } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { LineChart } from '../../components/charts/LineChart'
+import { Layer, useGetLayerStatsTxVolume } from '../../../oasis-indexer/api'
+import { chartUseQueryStaleTimeMs, durationToQueryParams } from '../../utils/chart-utils'
+import { DurationPills } from './DurationPills'
+import { ChartDuration } from '../../utils/chart-utils'
+
+export const TotalTransactions: FC = () => {
+  const { t } = useTranslation()
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+  const [chartDuration, setChartDuration] = useState<ChartDuration>(ChartDuration.WEEK)
+  const statsParams = durationToQueryParams[chartDuration]
+  const dailyVolumeQuery = useGetLayerStatsTxVolume(Layer.emerald, statsParams, {
+    query: {
+      keepPreviousData: true,
+      staleTime: chartUseQueryStaleTimeMs,
+    },
+  })
+
+  return (
+    <Card>
+      <CardHeader
+        action={!isMobile && <DurationPills handleChange={setChartDuration} value={chartDuration} />}
+        disableTypography
+        component="h3"
+        title={t('totalTransactions.header')}
+      />
+      {isMobile && (
+        <Box sx={{ mb: 5 }}>
+          <DurationPills handleChange={setChartDuration} value={chartDuration} />
+        </Box>
+      )}
```

Comment:
Noticed that `DurationPills` is duplicated for desktop and mobile. Could `CardHeader` be refactored, so that it changes `flex-direction` from `row` to `column`?

Reply:
Added `CardHeaderWithResponsiveActions` that will be used in another section later this week. I hit some mui ts limitation so I added a comment.

Switching between outlined and filled was causing 1px jumping. Added 'outlined-selected` variant.